### PR TITLE
[Quest] Reap What You Sow Convert to IF

### PR DESCRIPTION
**Pull request by jamesbradleym**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Converts quest `Reap What You Sow` to IF
Fixes quest priority with `Let Sleeping Dogs Lie`
Adds missing dialog

[Capture](https://www.youtube.com/watch?v=IFy1EklRnFA)
[Deathball Repeat Capture](https://www.youtube.com/watch?v=wTPGs2W4fyg)

## Steps to test these changes

Pre-Reqs:
`Let Sleeping Dogs Lie` not Accepted

Quest:
Speak to `Mashuu-Ajuu`(!pos 129 -6 167 238)
Turn in: `Sobbing Fungus` (!addItem 4565)
-- or --
Turn In: `Deathball` (!additem 4566)

repeat

### CODE REVIEW

File: scripts/quests/windurst/Reap_What_You_Sow.lua [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_COMPLETED and
+                not quest:getMustZone(player)
+        end,
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Akkeke'] = {
+                onTrigger = function(player, npc)
+                    return quest:getVar(player, 'Prog') == 1 and quest:event(468, 0, xi.item.POPOTO) or nil
```
Comment (claywar) [inline scripts/quests/windurst/Reap_What_You_Sow.lua]: I'm really not a fan of the sheer number of ternaries here.  I'd suggest breaking this out into another section

File: scripts/quests/windurst/Reap_What_You_Sow.lua [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                player:getQuestStatus(xi.questLog.WINDURST, xi.quest.id.windurst.LET_SLEEPING_DOGS_LIE) ~= xi.questStatus.QUEST_ACCEPTED and
+                not xi.quest.getMustZone(player, xi.questLog.WINDURST, xi.quest.id.windurst.LET_SLEEPING_DOGS_LIE)
+        end,
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Mashuu-Ajuu'] =
+            {
+                onTrigger = function(player, npc)
+                    if
```
Comment (claywar) [inline scripts/quests/windurst/Reap_What_You_Sow.lua:55]: formatting

File: scripts/quests/windurst/Reap_What_You_Sow.lua (hunk 1 above)
Comment (jamesbradleym) [reply scripts/quests/windurst/Reap_What_You_Sow.lua]: Cleaned up 👍🏻

**Comment by jamesbradleym** [comment]
Closing this PR to reduce open PRs and prevent staleness—will revisit.

**Pull request closed without merging**